docs/manual/jobs/ai-generated-runbooks.md

```diff
@@ -0,0 +1,88 @@
+# AI-Generated Runbooks (Beta)
+
+:::warning Beta Feature
+This feature is still in beta and therefore the behavior and performance of this feature is not guaranteed.
+:::
+
+**AI-Generated Runbooks** provides an AI-enhanced approach to creating new automation Jobs.  Through natural-language prompts, users can generate the baseline starting-point for the Job definition.
+
+![Job Preview](@assets/img/ai-gen-rba-job-preview.png)<br>
+
+The AI-Generated Runbooks feature helps new users get started with Runbook Automation. Through example prompts, new users quickly learn the types of use-cases that Runbook Automation is suited to address.
+
+![Example Prompts Provide Context to New Users](@assets/img/ai-gen-rba-example-prompt.png)
+
+<br>
+AI-Generated Runbooks also helps experienced authors of Jobs by reducing the time spent creating the Job definition and searching through third-party technical documentation.
+
+[comment]: <> (How to enable AI Gen RBA)
+## How to Enable AI-Generated Runbooks
+
+:::tip Product Availability
+The _AI-Generated Runbooks_ feature is only available in Runbook Automation.
+New users and current Process Automation customers are encouraged to [start a trial](https://www.pagerduty.com/sign-up/runbook-automation/) of Runbook Automation to try out the _AI-Generated Runbooks_ feature.
+:::
+
+To enable _AI-Generated Runbooks_:<br>
+1. Navigate to the AI Manager through the System Menu (gear icon in the upper-right):
+   ![AI Manager Menu](@assets/img/ai-manager-menu.png)
+2. Click the checkbox to `**Enable AI Generated Runbooks**`
+3. Select the projects where the feature should be enabled:
+   ![Enable in Projects](@assets/img/enable-ai-in-project.png)
+4. Click **Save**.
+
+:::warning First Time Configuration
+While the feature is still in beta, users will be asked to review and agree to the beta feature Terms & Conditions. Only **Add Admin** users are allowed to agree to the beta Terms & Conditions.  
+:::
+
+## Using AI-Generated Runbooks
+
+To use _AI-Generated Runbooks_:<br>
+
+1. Navigate to the **Jobs** tab.
+2. Click on **Generate Job**:
+   ![Generate Job](@assets/img/jobs-page-generate-job.png)
```

Comment:
Did you preview this?  sometimes we need <br/> entries after the images so it's not all squished together.

Reply:
yup - it looked fine in preview to me:

![Screen Shot 2023-08-31 at 7 55 32 AM](https://github.com/rundeck/docs/assets/11511251/2de819f3-40e8-4e30-a322-9c145a712894)